### PR TITLE
[DOCS-623] Partials - auto-hostendpoints-migrate

### PR DESCRIPTION
**Pull request by denis-alkhelali**

### CODE REVIEW

File: src/components/partials/auto-hostendpoints-migrate.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                    <p>Add any labels on existing all-interfaces host endpoints to their corresponding {props.orch} nodes. {props.prodname} manages labels on automatic host endpoints by syncing
+                    labels from their nodes. Any labels on existing all-interfaces host endpoints should be added to their respective nodes.
+                    For example, if your existing all-interface host endpoint for node <strong>node1</strong> has the label <strong>environment: dev</strong>, then you must add that same label to its node:</p>
+                    <pre><code>
+                        {props.orch === "OpenShift" ? (
+                            <span>oc label node node1 environment=dev</span>
+                        ) : (
+                            <span>kubectl label node node1 environment=dev</span>
+                        )}
+                    </code></pre>
```
Comment (doucol) [inline src/components/partials/auto-hostendpoints-migrate.js]: Is there a way to specify what kind of code block it is? Is there a docusaurus version of the codeblock we should be using here?

**Review by alexeymagdich-tigera — DISMISSED** [review]

File: src/components/partials/auto-hostendpoints-migrate.js (hunk 1 above)
Comment (doucol) [reply src/components/partials/auto-hostendpoints-migrate.js]: @denis-alkhelali I think you need to use this => 'import CodeBlock from "@theme/CodeBlock";'

**Review by alexeymagdich-tigera — DISMISSED** [review]

**Review by doucol — CHANGES_REQUESTED** [review]
Please convert the \<code\> blocks to \<CodeBlock\>

File: src/components/partials/auto-hostendpoints-migrate.js (hunk 1 above)
Comment (denis-alkhelali) [reply src/components/partials/auto-hostendpoints-migrate.js]: That's right :) wanted to have it merged and fix in current development branch
@doucol

**Review by doucol — APPROVED** [review]

**Pull request merged**